### DISCUSSION TITLE
Cannot add a suggestion for a greyed-out area in the Files / Files Changed tab

### DISCUSSION THREAD

**Discussion by electricjesus** [community/community#11159] [Pull Requests]
I can write a comment here fine: 
![image](https://user-images.githubusercontent.com/794828/152962143-c87e13d5-331c-476d-ac35-f7a9b78506b4.png)

or even in the black-background area above that (e.g. a non-change)

but sometimes i'd like to point out something on the grey area, too:

![image](https://user-images.githubusercontent.com/794828/152962667-84ac04e6-4c12-4891-973f-b86b57845f32.png)

But as you can see, the comment box plus button doesn't even appear. 

Also, is there a way to add suggestions that have line numbers as parameters e.g.:

![image](https://user-images.githubusercontent.com/794828/152963444-e21aa41a-937b-4b09-84e0-3664d0f618e3.png)

**Comment by oliver-sentianse** [comment]
This is from last february. Github no idea why this is happening? It is preventing from reviewing these lines, it is weird, I can't see any rhyme or reason to the lines that are grayed out.

**Comment by nalbion** [comment]
In the GitHub Advanced Security video they're able to create a ````suggestion` with "Outside changed files" targeting line 16 of package.json in GitHub comments. I'd love to know how this is done.

![image](https://github.com/community/community/assets/1866161/edf17b5f-da65-4d60-aafc-64e10a26c26c)

**Comment by ldynia** [comment]
I have the same problem. A huge portion of grey-outed code is folded, and I cannot comment on it. The reviewing experience is horrible!

**Comment by liampond** [comment]
Having the same problem. I wonder if it's bad practice because you're only supposed to review changes and not sections that have already been reviewed. But I'd still like to leave a comment!
